frontend/pages/_app.tsx

```diff
@@ -1,19 +1,33 @@
+import React, { useState, useEffect } from "react";
 import "../styles/globals.css";
 import "bootstrap/dist/css/bootstrap.css";
 import "@fortawesome/fontawesome-svg-core/styles.css";
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import React from "react";
 import PageNavbar from "../components/common/PageNavbar";
 import Footer from "../components/common/Footer";
 import DataState from "../context/DataContext/dataState";
 import { SpeechProvider } from "@speechly/react-client";
 import { MoralisProvider } from "react-moralis";
 import { NotificationProvider } from "web3uikit";
+import Loading from "../components/loadingAnimation";
+
 
 export default function App({ Component, pageProps }: AppProps) {
+  const [loading, setLoading] = useState(true);
+
+  // Simulate loading or asynchronous tasks
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 3000); // Simulating a 3-second loading time
+  }, []);
```

Comment:
I don't think we need this type of loading, if there is any specific reason for it, let me know